### ISSUE TITLE
Crashes at delegate and dataSource = self

### ISSUE THREAD

**Issue by kacak9229** [Yalantis/Koloda#340]
I'm using swift 3.2, I copied the exact same code from github, but keep getting crashes. The error showed at the viewDidLoad which delegate and dataSource reside

Thanks

```
import UIKit
import Koloda

class HomeViewController: UIViewController {
    
    let images = [#imageLiteral(resourceName: "bgGirl"), #imageLiteral(resourceName: "girlPassenger")]

    @IBOutlet weak var kolodaView: KolodaView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        
        kolodaView.delegate = self
        kolodaView.dataSource = self
        // Do any additional setup after loading the view.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    

    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        // Get the new view controller using segue.destinationViewController.
        // Pass the selected object to the new view controller.
    }
    */

}

extension HomeViewController: KolodaViewDelegate {
    func kolodaDidRunOutOfCards(_ koloda: KolodaView) {
        koloda.reloadData()
    }
    
    func koloda(_ koloda: KolodaView, didSelectCardAt index: Int) {
        UIApplication.shared.openURL(URL(string: "https://yalantis.com/")!)
    }
}


extension HomeViewController: KolodaViewDataSource {
    
    func kolodaNumberOfCards(_ koloda:KolodaView) -> Int {
        return images.count
    }
    
    func kolodaSpeedThatCardShouldDrag(_ koloda: KolodaView) -> DragSpeed {
        return .fast
    }
    
    func koloda(_ koloda: KolodaView, viewForCardAt index: Int) -> UIView {
        return UIImageView(image: images[index])
    }
    
    func koloda(_ koloda: KolodaView, viewForCardOverlayAt index: Int) -> OverlayView? {
        return Bundle.main.loadNibNamed("OverlayView", owner: self, options: nil)![0] as? OverlayView
    }
}

```

**Comment by irtaneja** [comment]
I am getting the same error. Crashes when I set datasource or delegate

**Comment by omerozk** [comment]
- On the storyboard click on your KolodaView
- On the right panel select "Identity Inspector"
- Under 'Custom Class' set the Module to Koloda and not to your target.
- (in case you didn't set it the view should be a KolodaView class (also under 'Custom Class'))

**Comment by jamesbaker1** [comment]
Same error for, even with setting my Module to Koloda.

**Comment by antonijap** [comment]
Setting my Module to Koloda solved my problems.

**Comment by cdiclemente** [comment]
After messing around with it a bit, it started working after I copied over the ExampleOverlayView.swift file from the example into my project.

After adding breakpoints, I noticed it crashed when trying to load the Nib "OverlayView". I believe it's because just copying the OverlayView.xib file into your project brings over an imageview that references ExampleOverlayView, but since the file is not in your project, it crashes.

For whatever reason, adding ExampleOverlayView.swift (as well as the rest of the example project files) into my project fixed this problem
